### ISSUE TITLE
websocket transport

### ISSUE THREAD

**Issue by davidmarkclements** [apparatus/mu#61]
Should be fairly straightforward to implement, one catch though - have alt browser client only incarnation that *doesnt* use streams

**Comment by mcollina** [comment]
it's really two transports. One client for Node and one for the Browser. But just one server.
How would auth work for browser clients?

**Comment by davidmarkclements** [comment]
yeah it's two transports, but they can be one module `mu-ws/index.js` and  `mu-ws/browser.js` with browser field set in `package.json`

re auth - maybe we should have an auth adapter that adds jwts over any transport?
